posthog/api/test/test_team.py

```diff
@@ -1604,6 +1608,66 @@ def _patch_linked_flag_config(
             assert response.status_code == expected_status, response.json()
             return response
 
+        def test_filters_null_values_from_app_urls(self):
```

Comment:
I'd switch this to a `parameterized.expand` test 
makes it way easier to see what is failing / passing
what conditions are covered

Reply:
💖  i love parameterized tests